deploy/harvester-load-balancer/crds/loadbalancer.harvesterhci.io_loadbalancers.yaml

```diff
@@ -27,10 +27,10 @@ spec:
       name: TYPE
       type: string
     - jsonPath: .status.internalAddress
-      name: INTERNALADDR
+      name: INTERNAL-ADDRESS
```

Comment:
Refer to the service and use `-` to separate the words. You can try `kubectl get svc -o wide`.